config/manifests/bases/cockroach-operator.clusterserviceversion.yaml

```diff
@@ -23,46 +23,94 @@ metadata:
     createdAt: CREATED_AT_PLACEHOLDER
     description: CockroachDB Operator
     operators.operatorframework.io/builder: operator-sdk-v1.1.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/cockroachdb/cockroach-operator
     support: Cockroach Labs
-  name: cockroach-operator.vX.Y.Z
+  name: cockroach-operator.v0.0.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: CrdbCluster is the CRD for the cockroachDB clusters API
+      displayName: CoachroachDB Operator
+      kind: CrdbCluster
+      name: crdbclusters.crdb.cockroachlabs.com.cockroachlabs.com
```

Comment:
This seems incorrect

Reply:
Indeed. I'll check it out

Reply:
As mentioned below, this is the current name, meaning leaving it like this is "correct"